osu.Game.Tests/Resources/countdown-settings.osu

```diff
@@ -0,0 +1,5 @@
+osu file format v14
+
+[General]
+Countdown: 2
+CountdownOffset: 3
```

Comment:
This beatmap file is implicitly used for test coverage in `LegacyBeatmapEncoderTest.TestEncodeDecodeStability` (it uses all `.osu` files from resources).